### PR TITLE
Adapt overridden kiali webroot to correct kiali service name

### PR DESCRIPTION
**Pull request by bashofmann**
The kiali service is not called `rancher-istio-kiali` anymore, but `kiali`. The overwritten `WEB_ROOT` url in https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-kiali-server/rancher-kiali-server.patch#L125 still references the old(?) service name. Also from my testing a `/kiali` suffix seems to be necessary now.

Because of the wrong `WEB_ROOT`, when you refresh the kiali page, you get a 404.

Addresses https://github.com/rancher/rancher/issues/29898.

### CODE REVIEW

**Comment by bashofmann** [comment]
I'm not sure if this is enough, or if I also need to bump some package or chart version, or if I need to merge this somewhere else.

**Review by brendarearden — APPROVED** [review]
LGTM

File: packages/rancher-kiali-server/package.yaml
Comment (aiyengar2) [inline packages/rancher-kiali-server/package.yaml:2]: @brendarearden I don't think it makes a difference w.r.t. the packaging of the chart since the `requirements.yaml` uses a local chart archive, but should we be updating https://github.com/rancher/charts/blob/70bcfcbee35caff374021bcc1fd0cb721f766640/packages/rancher-istio/charts/requirements.yaml#L6 with the correct `rancher-kiali-server` version (including the packageVersion)?

Comment (aiyengar2) [reply packages/rancher-kiali-server/package.yaml:2]: Note for future: This is a weird corner case of our charts scripts that needs to be addressed in https://github.com/rancher/rancher/issues/28730.

**Review by aiyengar2 — APPROVED** [review]
LGTM, let's wait on merging based on the response to my comment below. We may need to make a small change to bump the `rancher-kiali-server` version as well, just for documentation purposes.

File: packages/rancher-kiali-server/package.yaml
Comment (brendarearden) [reply packages/rancher-kiali-server/package.yaml:2]: I thought that the base version stayed the same, and we were appending the extra `000` or `001` on the end - is that not what the package version is for? Since we are just adjusting what we patch over the top of the 1.24.0 version, I think that requirements version should stay the same, but https://github.com/rancher/charts/blob/948c7b8f9a7a847eca3d680271ee0c2da11ddde5/packages/rancher-istio/charts/Chart.yaml#L16 would update... at least this was my understanding

Comment (aiyengar2) [reply packages/rancher-kiali-server/package.yaml:2]: Yeah I think that makes sense. That's how we've done it so far so probably that shouldn't change.

For this PR, I don't think we need to necessarily make that change (or rather shouldn't, to keep it consistent with the prior release) but it's definitely weird that our Helm packages point to a non-existent chart in the `requirements.yaml` when it should also be pointing to the actual version that the annotation points to. 

e.g. the file we point to in the `requirements.yaml` doesn't even exist in the `main` branch 

https://github.com/rancher/charts/blob/828fdd507ae0d5be0b6f017f39aff6742356c362/charts/rancher-istio/requirements.yaml#L6-L7

Something to fix in the charts script rewrite, but not something that needs to be addressed here 😅

**Comment by aiyengar2** [comment]
@bashofmann once you resolve conflicts, we can add the committer review and hopefully get this merged in!

File: packages/rancher-kiali-server/package.yaml
Comment (brendarearden) [reply packages/rancher-kiali-server/package.yaml:2]: yeah that is very weird - I see what you are saying now.

**Comment by brendarearden** [comment]
these changes were merged here https://github.com/rancher/charts/pull/848

**Pull request closed without merging**
